src/core/interfaces.d.ts

```diff
@@ -4,6 +4,10 @@ import Map from '../shim/Map';
 import WeakMap from '../shim/WeakMap';
 import { RegistryHandler } from './RegistryHandler';
 
+export type OmitProp<T, K extends keyof any> = Pick<T, Exclude<keyof T, K>>;
+export type Omit<T, K extends keyof T> = Pick<T, Exclude<keyof T, K>>;
```

Comment:
will this conflict with the global `Omit` type added in 3.5? I presume not but worth a check

Reply:
It shouldn't do, I did a quick check and there no complaints about `Omit`